draft-ietf-quic-qpack.md

```diff
@@ -821,62 +849,74 @@ have.  The smallest entry has empty name and value strings and has the size of
 decoder (see {{maximum-table-size}}).
 
 
-The decoder reconstructs the Largest Reference using the following algorithm:
+The decoder reconstructs the Required Insert Count using the following
+algorithm:
 
 ~~~
-   if LargestReference > 0:
-      LargestReference -= 1
+   if EncodedInsertCount == 0:
+      ReqInsertCount = 0
+   else:
+      InsertCount = EncodedInsertCount - 1
       CurrentWrapped = TotalNumberOfInserts mod (2 * MaxEntries)
 
-      if CurrentWrapped >= LargestReference + MaxEntries:
-         # Largest Reference wrapped around 1 extra time
-         LargestReference += 2 * MaxEntries
-      else if CurrentWrapped + MaxEntries < LargestReference
+      if CurrentWrapped >= InsertCount + MaxEntries:
+         # Insert Count wrapped around 1 extra time
+         ReqInsertCount += 2 * MaxEntries
+      else if CurrentWrapped + MaxEntries < InsertCount:
          # Decoder wrapped around 1 extra time
          CurrentWrapped += 2 * MaxEntries
 
-      LargestReference += TotalNumberOfInserts - CurrentWrapped
+      ReqInsertCount += TotalNumberOfInserts - CurrentWrapped
 ~~~
 
-TotalNumberOfInserts is the total number of inserts into the decoder's
-dynamic table.  This encoding limits the length of the prefix on
-long-lived connections.
+Where TotalNumberOfInserts is the total number of inserts into the decoder's
```

Comment:
I know that the actual intent is to continue the previous sentence (...using the following algorithm, where....), but this reads as a dangling phrase.  If you want the "where," maybe move the whole thing above the algorithm?